desktop/cmp/form/FormField.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import React, {Component} from 'react';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {PropTypes as PT} from 'prop-types';
+import {HoistInput} from '@xh/hoist/cmp/form';
+import {formGroup} from '@xh/hoist/kit/blueprint';
+import {div, span, hbox, hspacer} from '@xh/hoist/cmp/layout';
+import {throwIf} from '@xh/hoist/utils/js';
+import {isArray} from 'lodash';
+
+/**
+ * Wrapper around Blueprint's FormGroup component, which integrates
```

Comment:
I think comment should emphasize the relationship to @Field, and @FieldSupport, and the fact that it shows Label/Validation/Required state in a standard way.

I think the fact that it is a FormGroup is, at best, a secondary characteristic (if we publicize it at all)